tests/fib/test_fib.py

```diff
@@ -93,70 +101,72 @@ def test_fib(testbed, duthost, ptfhost, ipv4, ipv6, mtu):
                 log_file=log_file,
                 socket_recv_size=16384)
 
-@pytest.mark.parametrize("ipv4, ipv6", [pytest.param(True, True)])
-def test_hash(testbed, duthost, ptfhost, ipv4, ipv6):
-
-    config_facts = duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
 
+class Test_Hash():
```

Comment:
if you use camel notation, do not mix with understore. should be TestHash

Reply:
> if you use camel notation, do not mix with understore. should be TestHash

done.